steps/search_steps.js

```diff
@@ -1,43 +1,50 @@
 import { expect } from '@playwright/test'
 import { SearchPage } from "../pages/search_page";
-import { URN, DEFENDANT_NAME, DEFENDANT_DOB, ASN, NI_NUMBER } from '../config.js'
+// import { DEFENDANT_NAME, DEFENDANT_DOB, ASN, NI_NUMBER } from '../config.js'
 
 export class SearchSteps {
-  constructor(page) {
+  constructor(page, test_data) {
```

Comment:
Why are we suddenly using snake_case? Surely all this (testData, defendantName etc) should be camelCase?

Reply:
My rubyist part that resurfaces 😄